hack/version.sh

```diff
@@ -27,6 +27,9 @@
 #        "dirty" indicates source code changes after the git commit id
 #        "archive" indicates the tree was produced by 'git archive'
 #    KUBEVIRT_GIT_VERSION - "vX.Y" used to indicate the last release version.
+#    KUBEVIRT_SOURCE_DATE_EPOCH - unix timestamp.  Set to ' ' to generate using
```

Comment:
you can also set it to `date +%s` to make it more explicit that you want the current build time.